### ISSUE TITLE
cuda out of memory

### ISSUE THREAD

**Issue by mouawiya** [maudzung/Complex-YOLOv4-Pytorch#40]
hi, i was wondering if anyone knows how to solve this cuda out of memory problem, i've tried everything, lowering the batch size to 1, lowering the epoch number, nothing works, but at the same time i've tried other programs and they worked fine with cuda and with even normal batch=2, so i don't what to do next if anyone have any solution please help

**Comment by SinDongHwan** [comment]
Hi, @mouawiya.

Could you tell me your situations detail(environment, which program, etc)?

**Comment by mouawiya** [comment]
yes of course, i am runing the code on a windows 10, i have a 2GB nvidia graphic card (model GTX 950m), and i5 CPU, i am working with the Kitti dataset, the weird thing that i tried your other code SFA3D and it worked fine with batch size =2 , but this YOLOv4 it just cant seem to work for some reason i can't figure out

**Comment by SinDongHwan** [comment]
Hi, @mouawiya 

First, SFA3D uses the fpn_resnet_18 model and this repository uses the yolov4 model.
Two repositories uses different networks.

So fpn_resnet_18 is lighter than yolov4, so SFA3D would have worked.
Try loading the two models separately.

If you can't load the yolov4 model, you need to use a better graphics card to use this repository.

Good Luck~!!
